index.js

```diff
@@ -20,15 +20,13 @@ function init(editor) {
 			return;
 		}
 
-		const isTab = config.indent_style === 'tab' || !editor.softTabs;
```

Comment:
yeah, but this way it's more "expressive" and maybe feature proof (if case they add more options to the *indent_style* field).